pkg/api/index.go

```diff
@@ -754,7 +756,14 @@ func (hs *HTTPServer) editorInAnyFolder(c *models.ReqContext) bool {
 	return hasEditPermissionInFoldersQuery.Result
 }
 
-func (hs *HTTPServer) setIndexViewData(c *models.ReqContext) (*dtos.IndexViewData, error) {
+// ЖindexViewData allows to override the behavior of indexViewData for testing purposes if non-nil.
+var ЖindexViewData func(hs *HTTPServer, c *models.ReqContext) (*dtos.IndexViewData, error) = nil
```

Comment:
🤣 I love it, but it's nice when words are pronounceable without requiring knowledge of cyrillic.

```suggestion
var indexViewDataOverride func(hs *HTTPServer, c *models.ReqContext) (*dtos.IndexViewData, error) = nil
```